src/models/Offer.js

```diff
@@ -110,6 +110,14 @@ OfferSchema.query.current = function() {
     });
 };
 
+OfferSchema.query.activeOffers = function(canditateCompany) {
```

Comment:
As suggested, I renamed this query to `activeOffers`. However, this now returns a count, since I thought it made sense that countDoocuments was called in the query itself and not in the validator, since what we really want from the query is the count itself. Do you think it makes more sense this way, or should I change it?

Reply:
I think this query should return the values, as it may be useful in other scenarios. In the validator you can add the `countDocuments`. Do not forget to move this to the Company model instead